go/billing/models.py

```diff
@@ -176,6 +176,14 @@ class Meta:
         max_digits=10, decimal_places=3, default=Decimal('0.0'),
         help_text=_("The base cost per session in cents."))
 
+    session_unit_time = models.DecimalField(
+        max_digits=10, decimal_places=3, default=Decimal('20.0'),
+        help_text=_("The time of one billed unit (in seconds)."))
```

Comment:
Ya, thinking about it a bit more, having the session unit time's be changeable per message cost maybe isn't as complicated as I thought.

@hodgestar unless I misunderstood you, I think your preference was to use a static unit time of 20 seconds, instead of allowing each a message cost to have its own configurable session unit time. If this is the case, could you explain why?